code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -611,8 +634,9 @@
 		if(stolen_item)
 			C.unEquip(stolen_item)
```

Comment:
No check on this if it actually succeeds or not.
Nodrop items cannot be taken from somebody and those will break here